NSS-Day-Cohort-53/giffygram-rantingtyras
# Jb dark mode

```text
#### Changes Made
1. Added click event for dark mode
2. added dark mode icon/button in nav bar
3. added dark mode to main page, profile, message page, message/post forms, header, footer
4. changed link color to be better for dark mode.
​
#### Steps to Review
1. Checkout this branch locally.
    ```
    git fetch --all
    git checkout jb-dark-mode
    ```
2. Open a new Terminal tab (⌘T) and navigate to the server directory.
3. Test app functionality.
    > Instructions for how reviewer can test functionality, and detailed description of what the expected outcome is.
    > Example: dark mode button should work and persist over all pages
4. View code file.
    > Confirm file modifications are present as indicated above.
    > Confirm no unused code or extraneous comments exist.
```
